src/Kdyby/Translation/Translator.php

```diff
@@ -143,7 +143,7 @@ public function translate($message, $count = NULL, $parameters = array(), $domai
 		$parameters = $tmp;
 
 		if ($count !== NULL && is_scalar($count)) {
-			return $this->transChoice($message, preg_replace('~[^0-9,.]~', '', $count), $parameters + array('%count%' => $count), $domain, $locale);
+			return $this->transChoice($message, preg_replace('~[^0-9,.\-]~', '', $count), $parameters + array('%count%' => $count), $domain, $locale);
```

Comment:
I'm not sure why is this needed anyway. The only next usage is [here](https://github.com/symfony/Translation/blob/master/Translator.php#L237), casting value to integer and thanks to previous line in this file we can be sure that typecasting will never fail.
To be honest, this was only hotfix so I could continue in work, so I did not think about this very much :)

So:
 - if this is not needed, should I update PR with preg_replace removed?
 - if this is needed, would you please point me why & how to update PR to make this OK?

Thank you for your time!